### DISCUSSION TITLE
Apple sign-in

### DISCUSSION THREAD

**Discussion by 1MM0R7AL5** [PlayCover/PlayCover#1730] [Q&A] [answered]
Hey, this is my first time starting a discussion, so please be kind (I'm also diagnosed with autism, so keep that in mind). Would it be possible for app logins to work with Apple sign-ins? I'm not sure if it's possible since I'm guessing PlayCover only emulates the games themselves, but I'd love to see this implemented. An example of this type of game's _Swordash_. In more outlandish territory, I'd like Apple Arcade games to not crash. I've been trying to play the discontinued game _World of Demons_ for nostalgia purposes, and every Apple Arcade game crashes. _League of Legends: Wild Rift_ also crashes when I log into my Riot account.

**Comment by Catta1997** [comment]
There are a lot of posts in official discord, some are support it and some not. Afaik should work if you disable SIP, AMFI and PlayChain but never tried (I log in with mail/pass every time)

**Reply by 1MM0R7AL5** [reply]
What are AMFI and SIP? I don't understand those acronyms.

**Reply by Catta1997** [reply, marked as answer]
> What are AMFI and SIP? I don't understand those acronyms.

SIP -> System Integrity Protection
AMFI -> Apple Mobile File Integrity
This are 2 security feature of macOS, its not a toggle in setting but you need to do some steps in terminal. I suggest you to search on google "how to disable x", since there are a lot of tutorial and I never did it myself. If your game support login with mail/password I will not suggest you to disable them since are security things and I suggest you to login with mail. (PlayChain can be disabled on app settings (on PlayCover) on "bypass" tab)

**Reply by 1MM0R7AL5** [reply]
Got it, thanks!
